lusidtools/cocoon/cocoon.py

```diff
@@ -749,15 +785,37 @@ def load_from_data_frame(
         column_list = list(set([item for sublist in column_list for item in sublist]))
         data_frame = strip_whitespace(data_frame, column_list)
 
-    # Check for and create missing property defintions
-    data_frame = cocoon.properties.create_missing_property_definitions_from_file(
-        api_factory=api_factory,
-        properties_scope=properties_scope,
-        file_type=file_type,
-        data_frame=data_frame,
-        property_columns=property_columns,
-        domain_lookup=domain_lookup,
-    )
+    open_api_types = getattr(lusid.models, domain_lookup[file_type]["top_level_model"]).openapi_types
+
+    if "sub_holding_keys" in open_api_types.keys() and "dict" in open_api_types["sub_holding_keys"]:
```

Comment:
I think it would be useful to add a comment to say what this is doing

Reply:
Added